tests/models/test_loss.py

```diff
@@ -0,0 +1,61 @@
+from predict_pv_yield.models.loss import WeightedLosses
+import torch
+import pytest
+
+
+def test_weight_losses_weights():
+    forecast_length = 2
+    w = WeightedLosses(forecast_length=forecast_length)
+
+    assert w.weights.cpu().numpy()[0] == pytest.approx(2 / 3)
+    assert w.weights.cpu().numpy()[1] == pytest.approx(1 / 3)
+
+
+def test_mae_exp():
+    forecast_length = 2
+    w = WeightedLosses(forecast_length=forecast_length)
+
+    output = torch.Tensor([1, 3])
+    target = torch.Tensor([1, 5])
+
+    loss = w.get_mae_exp(output=output, target=target)
+
+    assert loss == pytest.approx(2 / 3)  # (1-1)*2/3 + (5-3)*1/4
```

Comment:
I like that you 'show your working' in the comment :)

Reply:
thanks